assignments/HW6.md

```diff
@@ -0,0 +1,223 @@
+
+### 3. Plan the task
+
+Once you have selected a project and task(s), break them down into subtasks, consider their priority and assign them to each team member. Identify a set of tasks as your core goal for this project, and another set of tasks as stretch goals.
+
+You are expected to achieve your core goal for this project, and stretch goals as much as possible. We will work with you to adjust your goals during the check-in to ensure that they are appropriate your team size and timeframe.
+
+As per previous homeworks, plan before you start coding. You should identify risks and requirements, and develop a collaboration plan and schedule. 
+
+### 4. Perform the task
+
+You should perform adequate quality assurance activities when writing code.
+
+1. **Take further steps to understand the project’s code.**  
+You might find it useful to engage in intra-team discussions using static or dynamic diagrams. You might also find it useful to elicit feedback on your ideas by communicating with members of the open-source community.
+
+1. **Submit your changes to the project**  
+Create any necessary documentation to enable acceptance of your code. Common contribution mechanisms include pull requests, emails to a project lead, or discussion board posts. You may also need to update the bug database. The project owners will review and evaluate your pull request, and you might need more work to get it approved.  
+**You are required to submit your work to the open-source project**
+
+1. **BONUS: Get your changes accepted**  
+You will get a bonus **20 (6%) bonus points** upon acceptance. If your code is accepted after the homework deadline but before the final exam, inform the course staff.
+
+### 5. Report, Presentation and reflection
+You will report on your project and task selection, work, and experience in several ways (see below). This will include a group presentation to the class.
+
+## Deliverable Details
+
+### Part A: Project & Tasks Selection Check-In 
+
+**Points**: 80  
+**Presentation Slide Deck Due Date**: 29th November 2022, 2359  
+**Presentation Date**: 30th November 2022 / 2nd December 2022 (Recitation after Thanksgiving) 
+
+The recitation after thanksigiving will be dedicated to an **in-person** group presentation about your project and task selection.
+
+- Presentations are **6 minutes, with 3 minutes feedback and Q&A**.
+- The order of presentations will be randomly determined.
+- Participation from all team members during the presentation is **required**.
+- For full credit, you would have to be on time for the presentation session (within 5 minutes of recitation start time). If you are unable to attend in-person, you have to send a slack message to your TA with justifications **24 hours before the start of your recitation** else you will receive **no credit** for this part of the assignment. You will then present by recording your segment of the presentation, which your teammate will play during the presentation.
```

Comment:
I changed this to make it such that people who can't make it does a recording instead.